File: app/src/main/java/com/adrianiglesia/atqr/Services/ApiServices.java
package com.adrianiglesia.atqr.Services;

import com.adrianiglesia.atqr.Model.LoginBody;
import com.adrianiglesia.atqr.Model.ResponseService;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface ApiServices {

    @POST("student/testlogin")
    Call<ResponseService> userLogin(@Body LoginBody body);
}

File: app/src/main/java/com/adrianiglesia/atqr/Services/ApiServices.kt
package com.adrianiglesia.atqr.Services

import com.adrianiglesia.atqr.Model.LoginBody
import com.adrianiglesia.atqr.Model.ResponseService

import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST

interface ApiServices {

    @POST("student/testlogin")
    fun userLogin(@Body body: LoginBody): Call<ResponseService>
}
